### ISSUE TITLE
state of block can not be changed by clicking

### ISSUE THREAD

**Issue by rallek** [zikula/core#2734]
In blocks menu I tried to deactivate a block by clicking on the ``active`` button. But nothing happens.

My site is in ``debug:true`` mode.

My workaround: Create a blockzone with name ``deactivated`` :-)

**Comment by craigh** [comment]
activating/deactivating blocks works great for me.

**Comment by rallek** [comment]
not for me. I tried in Firefox and Chrome. In both cases it do nothing. The 'bubble'' with the text ``Click to deactivate block`` is not disappearing until I click somewhere else.

Right click tells me the link is going to ``http://142.koester123.de/blocks/admin/view#``

It is an install from ci build 48 with overwriting the files by ci build 52

**Comment by craigh** [comment]
There is a problem with the loading of JS on your site.

![screen shot 2016-01-10 at 2 27 15 pm](https://cloud.githubusercontent.com/assets/350048/12223481/e0618714-b7a6-11e5-9569-60a24672fd97.png)

![screen shot 2016-01-10 at 2 27 33 pm](https://cloud.githubusercontent.com/assets/350048/12223482/e943a90c-b7a6-11e5-96a4-5c9a39766063.png)

not sure what the problem is ATM.

**Comment by craigh** [comment]
It could be a routing problem.

**Comment by rallek** [comment]
to make shure nothing is misconfigured I will set up the site completely from scratch. I will come back with a comment.

**Comment by rallek** [comment]
absolutely fresh isntallation of ci build 52 do have still the same behavior

**Comment by craigh** [comment]
may be related to #2672

**Comment by rallek** [comment]
Multilingual is not switched on in the current scenario

**Comment by craigh** [comment]
I'm having these same javascript problems now locally also.

**Comment by craigh** [comment]
@phaidon this is a problem that very recently cropped up. it seems to be related somehow to your "special" version of "bootstrap-plus/bootstrap-jqueryui" but I've no real way to be sure. I get the errors above (screenshots) on a clean install of the current core in both the Blocks admin list view and in the "new block" page. Can you look into it?

also - I noticed your version of the lib is using jQueryUI 1.10.4 and the newest version is 1.11.4. For some reason we have both of these in the core.

**Comment by craigh** [comment]
![screen shot 2016-01-15 at 4 29 24 pm](https://cloud.githubusercontent.com/assets/350048/12365818/337149a6-bba5-11e5-8992-ed9aafdb3aa2.png)

**Comment by craigh** [comment]
So I've learned that the problem is that jQuery has been updated to 2.2.0 (in the background via composer) and this causes some BC break. So I guess we can fix our dependency on 2.1.* and it should work ok. Now I have to figure how to do that in our jumbled mess of front-end dependency management (or lack of it). It has something to do with https://github.com/components/jquery and https://github.com/zikula/JQueryBundle
